src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -219,16 +217,18 @@ export default class SummaryBarGraph extends React.Component<ISummaryBarGraphPro
                                 }
                                 return obj;
                             }, {} as any)
-                            return _.reduce(alterationCounts, (arr, value:{count:number, backgroundColor: string}, label) => {
+                            const alterationLabels = _.reduce(alterationCounts, (arr, value:{count:number, backgroundColor: string}, label) => {
                                 if (value.count) {
                                     arr.push({
                                         text: this.getLegendNames(label),
-                                        fillStyle: value.backgroundColor,
-                                        index: counter ++
+                                        fillStyle: value.backgroundColor
                                     });
                                 }
                                 return arr;
                             }, [] as any);
```

Comment:
id you type the accumulator passed to reduce, the ts will consider alterationLabels type and you shouldn't need to type of params passed to sort below